atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -126,6 +131,28 @@ def generate_auto_labels(self, auto_labels):
         docker_info = self.tasker.get_info()
         generated['com.redhat.build-host'] = docker_info['Name']
 
+        info_url = self.info_url_format
+        labels = Labels(self.labels)
+        find_subs = re.finditer('{([^}]+)}', info_url)
```

Comment:
well problem is that I don't have such a list, as there could be used mixed (old/new) labels, so I need to use Labels